conbench/app/_plots.py

```diff
@@ -471,6 +471,8 @@ def time_series_plot(
 
     unit = samples[0].unit
     with_dist = [s for s in samples if s.zscorestats.rolling_mean]
+    inliers = [s for s in samples if not s.zscorestats.is_outlier]
+    outliers = [s for s in samples if s.zscorestats.is_outlier]
```

Comment:
love the readability of these list comprehensions; that also shows the value of recent refactorings